### ISSUE TITLE
DIS7 build error on Windows

### ISSUE THREAD

**Issue by leif81** [open-dis/open-dis-cpp#38]
Building results in what looks like the same 97 error for the OpenDIS7 project - a bunch of LNK2019 & LNK2001 errors.

MS Windows VM - https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/

_Originally posted by @rodneyp290 in https://github.com/open-dis/open-dis-cpp/pull/37#issuecomment-606974306_

**Comment by rodneyp290** [comment]
The errors are related to #18, as the OpenDIS7 considers `utils/*.cpp` as source files that include some OpenDIS 6 (`dis6/*.h`) files. 
I was able to build the OpenDIS7 with the patches below for CMake, and Premake, although that is just building it and I'm not sure it works as intended.
CMake patch:
```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index bc44ff0..0a62c03 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,7 +19,7 @@ target_compile_definitions(OpenDIS6 PRIVATE EXPORT_LIBRARY)
 # create list of DIS7 source files
 file(GLOB DIS7_SOURCES
   "src/dis7/*.cpp"
-  "src/utils/*.cpp"
+  "src/utils/DataStream.cpp"
 )
 # Define ExampleSender Executable
 add_library(OpenDIS7 SHARED ${DIS7_SOURCES})
```
Premake patch
```diff
diff --git a/premake5.lua b/premake5.lua
index 1d9d745..7f7291d 100644
--- a/premake5.lua
+++ b/premake5.lua
@@ -47,8 +47,9 @@ project "OpenDIS7"
   files {
     "src/dis7/*.h",
     "src/dis7/*.cpp",
-    "src/utils/*.h",
-    "src/utils/*.cpp"
+    "src/utils/Endian.h",
+    "src/utils/DataStream.h",
+    "src/utils/DataStream.cpp"
   }
 
 project "ExampleSender"
```

**Comment by rodneyp290** [comment]
If you want to bundle all the Utils into OpenDIS7 library, then the above isn't a suitable solution as it only includes the DataStream class, and Endian enum that it requires.
Just noticed as I'm typing that the Endian enum still includes the `dis6/msLibMacro.h`, however that isn't a problem because the `dis*/msLibMacro.h` files are exactly the same, and shouldn't interfere with each other. Might be a possible to consolidate them.

**Comment by leif81** [comment]
This should be fixed by the recently merged #76 

I'm going to close this issue but reopen if you're still seeing an issue.
